test-lit/semantics/allocate10.f90

```diff
@@ -3,6 +3,9 @@
 !TODO: mixing expr and source-expr?
 !TODO: using subcomponent in source expressions
 
+! RUN: not %flang -fdebug-resolve-names -fparse-only %s 2>&1 | FileCheck --check-prefixes=C942a,ERROR %s
```

Comment:
what is the use of C942a prefix?

Reply:
None, unfortunately the prefix finder picks up anything prefix-like, and these standard references (I assume), are sometimes formatted like prefixes, and not like comments (like they should be)

Reply:
So, I'm going to take a fairly big assumption that prefix-like string that starts with a C, followed by a number, is not a prefix. From the prefixes I've seen, I don't think this is a problem, but could need further refinement later on.